src/main/java/com/teragrep/rlp_01/RelpFrameTX.java

```diff
@@ -92,26 +92,21 @@ private void putData(ByteBuffer dst) {
      * Writes a HEADER part of the RELP message to the byte buffer.
      *
      * @param dst
-     * @throws UnsupportedEncodingException
      *  Shouldn't happen for US-ASCII..
      */
-    private void putHeader(ByteBuffer dst) throws UnsupportedEncodingException {
-        dst.put(Integer.toString(this.transactionNumber).getBytes("US-ASCII"));
+    private void putHeader(ByteBuffer dst) {
+        dst.put(Integer.toString(this.transactionNumber).getBytes(StandardCharsets.US_ASCII));
         dst.put((byte)' ');
-        dst.put(this.command.getBytes("US-ASCII"));
+        dst.put(this.command.getBytes(StandardCharsets.US_ASCII));
         dst.put((byte)' ');
-        dst.put(Integer.toString(this.dataLength).getBytes("US-ASCII"));
+        dst.put(Integer.toString(this.dataLength).getBytes(StandardCharsets.US_ASCII));
     }
 
     /**
      Override for toString method. Returns the entire RELP message formatted with spaces.
      */
     @Override
     public String toString() {
-        try {
-            return this.transactionNumber + " " + this.command + " " + this.dataLength + " " + (this.data != null ? new String(this.data, "UTF-8") : "");
-        } catch (UnsupportedEncodingException e) {
-            throw new RuntimeException(e);
-        }
+        return this.transactionNumber + " " + this.command + " " + this.dataLength + " " + (this.data != null ? new String(this.data, StandardCharsets.UTF_8) : "");
```

Comment:
i think this should return rather null instead of empty string in case data is null, however maybe worth of another issue